ironfish/src/network/peerNetwork.ts

```diff
@@ -1198,13 +1198,15 @@ export class PeerNetwork {
       return
     }
 
+    const firstPeer = this.blockFetcher.firstSeenBy(block.header.hash)
```

Comment:
Think it would make sense to move this down to right above the telemetry line? Since for now we're only using it for that